src/main/java/org/wso2/carbon/module/swiftiso20022/validation/common/ValidatorContext.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.swiftiso20022.validation.common;
+
+/**
+ * Context class to hold the request attributes for validation.
+ */
+public class ValidatorContext {
+
+    private final String fieldName;
+    private final String fieldDisplayName;
+    private int fieldLength;
```

Comment:
Shall we remove the `final` keyword and add setter methods for these fields?

Reply:
From the current perspective, we don't need the setter methods.